### DISCUSSION TITLE
数据集下载地址

### DISCUSSION THREAD

**Discussion by Javesun99** [apachecn/MachineLearning#642] [Q&A]

**Comment by jiangzhonglian** [comment]
> github 下载地址

* 资料来源: Machine Learning in Action(机器学习实战-个人笔记)
* 统一数据地址: <https://github.com/apachecn/data>
* 书籍下载地址: <https://github.com/apachecn/data/tree/master/book>
* 机器学习下载地址: <https://github.com/apachecn/data/tree/master/机器学习>
* 深度学习数据地址: <https://github.com/apachecn/data/tree/master/深度学习>
* 推荐系统数据地址: <https://github.com/apachecn/data/tree/master/推荐系统>

---

> 更新时间 2024-02-05

- 代码（在 src 目录下）下载地址：[AiLearning-master-20200807.zip](https://github.com/apachecn/AiLearning/files/5040842/AiLearning-master-20200807.zip)
- 数据下载地址：<http://82.156.167.139:5244/d/大肥羊学校/数据/机器学习实战-数据.zip>

**Comment by Chochenxi** [comment]
感谢~





------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"片刻"<notifications@github.com&gt;;
发送时间:&nbsp;2020年3月31日(星期二) 晚上6:15
收件人:&nbsp;"apachecn/AiLearning"<AiLearning@noreply.github.com&gt;;
抄送:&nbsp;"Subscribed"<subscribed@noreply.github.com&gt;;
主题:&nbsp;Re: [apachecn/AiLearning] 可以提供一下数据集吗？谢谢 (#580)





  
资料来源: Machine Learning in Action(机器学习实战-个人笔记)
 
统一数据地址: https://github.com/apachecn/data
 
书籍下载地址: https://github.com/apachecn/data/tree/master/book
 
机器学习下载地址: https://github.com/apachecn/data/tree/master/机器学习
 
深度学习数据地址: https://github.com/apachecn/data/tree/master/深度学习
 
推荐系统数据地址: https://github.com/apachecn/data/tree/master/推荐系统

**Comment by wizardforcel** [comment]
@jiangzhonglian 以后学习怎么用百度秒传和阿里秒传
